jobs/kubernetes-system-specs/templates/config/kubedns-rc.yml

```diff
@@ -30,11 +39,17 @@ spec:
         k8s-app: kube-dns
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
     spec:
+      tolerations:
+      - key: "CriticalAddonsOnly"
+        operator: "Exists"
+      volumes:
+      - name: kube-dns-config
+        hostPath:
```

Comment:
The ConfigMap in the 1.6.x configuration was optional, and was [initially blank](https://github.com/kubernetes/kubernetes/blob/v1.6.1/cluster/addons/dns/kubedns-cm.yaml). 
We do not use it in our setup, and it is unrelated to the `hostPath` change.